brave-kafka/src/main/java/smartthings/brave/kafka/consumers/BaseTracingConsumerInterceptor.java

```diff
@@ -74,11 +87,8 @@
         TracedConsumerRecord<K, byte[]> tracedConsumerRecord = getTracedConsumerRecord(record);
         TraceContextOrSamplingFlags traceContextOrSamplingFlags =
           tracedConsumerRecord.traceContextOrSamplingFlags;
-        TraceContext ctx = traceContextOrSamplingFlags.context();
```

Comment:
https://github.com/openzipkin/brave/blob/master/instrumentation/kafka-clients/src/main/java/brave/kafka/clients/KafkaTracing.java#L118

though this file itself possible is delete worthy, an aside to consider what needs to happen now vs later